public/js/monitor.js

```diff
@@ -326,82 +326,82 @@ function recruitmentLogic() {
-          return;
-        }
-        ga("send", "event", "Waitlist Test", "Engage", relayCta.dataset.analyticsLabel);
-      });
-    });
-  }
-}
+// function addWaitlistSignupButtonListeners() {
+//   document.querySelectorAll(".relay-sign-up-btn").forEach(btn => {
+
+//     btn.addEventListener("click", async (e) => {
+//       const relayEndpoint = new URL("/join-waitlist", document.body.dataset.serverUrl);
+//       const signUpCallout = document.querySelector(".relay-sign-up");
+
+//       signUpCallout.classList.add("sending-email");
+//       try {
+//         const response = await fetch(relayEndpoint, {
+//           headers: {
+//             "Content-Type": "application/json; charset=utf-8",
+//           },
+//           mode: "same-origin",
+//           method: "POST",
+//           body: JSON.stringify({ "emailToAdd": "add-user-email" }),
+//         });
+//         if (response && response.status === 200) {
+//           setTimeout(() => {
+//             signUpCallout.classList.add("email-sent");
+//             signUpCallout.classList.remove("sending-email");
+//           }, 500);
+//         }
+//       } catch (e) {
+//         // we need error messaging
+//       }
+//     });
+//   });
+// }
+
+// function addWaitlistObservers() {
+
+//   const privateRelayCtas = document.querySelectorAll(".private-relay-cta");
+
+//   if (privateRelayCtas.length === 0) {
+//     return;
+//   }
+//   const availableIntersectionObserver = ("IntersectionObserver" in window);
+//   const gaAvailable = typeof (ga) !== undefined;
+
+
+//   if (availableIntersectionObserver && gaAvailable) {
+//     const sendWaitlistViewPing = elemData => {
+//       if (elemData.userIsSignedUp === "true") {
+//         return;
+//       }
+//       ga("send", "event", "Waitlist Test", "View", elemData.analyticsLabel);
+//     };
+//     const onRelayCtasComingIntoView = (entries, observer) => {
+//       entries.forEach(entry => {
+//         if (entry.isIntersecting) {
+//           sendWaitlistViewPing(entry.target.dataset);
+//           observer.unobserve(entry.target);
+//         }
+//       });
+//     };
+//     const observer = new IntersectionObserver(onRelayCtasComingIntoView, { rootMargin: "-50px" });
+
+//     privateRelayCtas.forEach(relayCta => {
+//       observer.observe(relayCta);
+//       relayCta.addEventListener("click", (e) => {
+//         if (relayCta.href) {
+//           if (typeof (ga) !== "undefined") {
+//             // Recheck if the user is on strict-mode and only block the click default action if GA is available
+//             e.preventDefault();
+//             ga("send", "event", "Waitlist Test", "Engage", relayCta.dataset.analyticsLabel, {
+//               "hitCallback": window.location.href = relayCta.href,
+//             });
+//           }
+//           return;
+//         }
+//         ga("send", "event", "Waitlist Test", "Engage", relayCta.dataset.analyticsLabel);
+//       });
+//     });
+//   }
+// }
```

Comment:
question: Should we just go ahead an remove this code? Or stash it in a .js file that isn't used or referenced?

Reply:
Yeah... I was saving it in order to follow the crumbs and delete all server-side portions of it post-data-removal pilot... I _think_ it might be tied in to a reused route in the pilot, and didn't want to deal with merge conflicts.  I could certainly add a "TODO"... 😬